### PR TITLE
BUGFIXES 

### PR DESCRIPTION
**Pull request by dbulatovicx32**
[KNOWAGE-7976](https://production.eng.it/jira/browse/KNOWAGE-7976)
[KNOWAGE-7997](https://production.eng.it/jira/browse/KNOWAGE-7997)
[KNOWAGE-7999](https://production.eng.it/jira/browse/KNOWAGE-7999)

More info in individual commits per issue number.

### CODE REVIEW

File: src/modules/documentExecution/dashboard/generalSettings/background/DashboardBackground.vue [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <template>
     <div class="p-d-flex p-flex-column kn-flex p-mr-3 p-my-3 dashboard-card-shadow kn-overflow dashboard-scrollbar">
         <label class="kn-material-input-label p-m-3"> {{ $t('dashboard.widgetEditor.background') }}</label>
-
+        {{ background }}
```
Comment (Redjaw) [inline src/modules/documentExecution/dashboard/generalSettings/background/DashboardBackground.vue:4]: is this print (and the console log at line 56) intentional for debugging purpose or can we remove it?

**Review by Redjaw — CHANGES_REQUESTED** [review]

File: src/modules/documentExecution/dashboard/generalSettings/background/DashboardBackground.vue (hunk 1 above)
Comment (dbulatovicx32) [reply src/modules/documentExecution/dashboard/generalSettings/background/DashboardBackground.vue]: That one escaped me, removed!

Comment (Redjaw) [reply src/modules/documentExecution/dashboard/generalSettings/background/DashboardBackground.vue]: thank you!

File: src/modules/documentExecution/dashboard/generalSettings/background/DashboardBackground.vue [hunk 2]
```diff
@@ -52,9 +51,12 @@ export default defineComponent({
             this.dashboard = this.dashboardModelProp
             if (!this.dashboard.configuration?.background) this.dashboard.configuration.background = { sheetsBackgroundColor: '', imageBackgroundUrl: '', imageBackgroundSize: '' } as IBackground
             this.background = this.dashboard.configuration.background as IBackground
+
+            console.log('this.dashboard.configuration.background', this.dashboard.configuration.background)
```
Comment (Redjaw) [inline src/modules/documentExecution/dashboard/generalSettings/background/DashboardBackground.vue:55]: remember to remove those after testing

**Review by Redjaw — APPROVED** [review]

**Pull request merged**